packages/react/src/MegaMenu/MegaMenu.tsx

```diff
@@ -26,7 +26,7 @@ const MegaMenuListCategory = forwardRef(
     { children, className, ...restProps }: PropsWithChildren<HTMLAttributes<HTMLDivElement>>,
     ref: ForwardedRef<HTMLDivElement>,
   ) => (
-    <div {...restProps} ref={ref} className={clsx('amsterdam-mega-menu__list-wrapper', className)}>
+    <div {...restProps} ref={ref} className={clsx('amsterdam-mega-menu__list-category', className)}>
```

Comment:
I guess we can move this subcomponent into its own file and add our default unit tests, as we did in #1050?

Reply:
Nice catch though!